website/docs/api/topBar-searchBar.mdx

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
@mateioprea We should probably include the `Platform` column to indicate all of these options are iOS specific.

Reply:
Done